src/CloudFoundry.VisualStudio/CloudFoundry.VisualStudioPackage.cs

```diff
@@ -159,6 +159,7 @@ private void ButtonBuildAndPushProjectChangeHandler(object sender, EventArgs e)
         {
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Catch general exception to prevent VS crash")]
```

Comment:
The exception message thrown is incorrect. Many other things can go wrong here, with an invalid error message show to the user.
The publish profile load method should catch serialization errors and throw the appropriate error (as our own exception type - e.g. VisualStudioException) and only catch that here.